admin/company/settings/sso-configuration.md

```diff
@@ -12,8 +12,8 @@ Follow the steps on this page to configure SSO for your company.
 
 1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
```

Comment:
Given this is a repeat of what's on the domain management page, perhaps it'd be better to just link to that page? I'd keep the header and then something like 'Follow the instructions provided in Domain management'